servarr/values.yaml

```diff
@@ -6,6 +6,10 @@ global:
   # -- (string) Insert your storage class here, e.g.: &storageClassName network-block. Do not remove the `&storageClassName` anchor!
   # @section -- Global
   storageClassName: &storageClassName "network-block"
+  # -- (string) Insert your ingress class here, e.g.: &ingressClassName nginx. Do not remove the `&ingressCassName` anchor!
```

Comment:
Since we found out that an empty or blank value will produce a "null" ingress class name in the ingress resource, can we explicitly state this in the comment (which will be included in the chart README)? In this way we let users know they must write something.

Reply:
Done!